examples/federated-database-instance/main.tf

```diff
@@ -56,9 +56,11 @@ EOF
 resource "mongodbatlas_federated_database_instance" "test" {
   project_id = var.project_id
   name       = var.name
-  aws {
-    role_id        = mongodbatlas_cloud_provider_access_authorization.auth_role.role_id
-    test_s3_bucket = var.test_s3_bucket
+  cloud_provider_config {
+    aws {
+      role_id        = mongodbatlas_cloud_provider_access_authorization.auth_role.role_id
+      test_s3_bucket = var.test_s3_bucket
+    }
```

Comment:
No, it is not a breaking change as this resource is not released yet. It will be the first release in v1.10.0.

Right now `aws` is at the top level but, based on this https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1173#discussion_r1220613438, we want to move it inside `cloud_provider_config`

Reply:
Ahh excellent, *phew* :) Carry on!